gammapy/catalog/fermi.py

```diff
@@ -56,22 +56,226 @@ class SourceCatalogObject3FGL(SourceCatalogObject):
+               ' source TS > 25, or the band with highest TS if all are < 25. theta_ref is set to 2.17 degrees (FWHM)'
+               ' below 300 MeV, 1.38 degrees between 300 MeV and 1 GeV, 0.87 degrees between 1 GeV and 3 GeV, 0.67'
+               ' degrees between 3 and 10 GeV and 0.45 degrees about 10 GeV (2*r_68).',
+            6: 'On top of an interstellar gas clump or small-scale defect in the model of diffuse emission. This flag '
+               'is equivalent to the "c" suffix in the source name.',
+            7: 'Unstable position determination; result from gtfindsrc outside the 95% ellipse from pointlike.',
+            9: 'Localization Quality > 8 in pointlike (see Section 3.1 in catalog paper) or long axis of 95% ellipse >'
+               ' 0.25.',
+            10: 'Spectral Fit Quality > 16.3 (see Equation 3 in 2FGL catalog paper).',
+            11: 'Possibly due to the Sun (see Section 3.6 in catalog paper).',
+            12: 'Highly curved spectrum; LogParabola beta fixed to 1 or PLExpCutoff Spectral Index fixed to 0 (see '
+                'Section 3.3 in catalog paper).'
+        }
+        ss += '{:<30s} : {}\n'.format('Other flags', flagMessage.get(d['Flags'], 'N/A'))
+
+        return ss
+
+    def _info_position(self):
+        """Print position info."""
+        d = self.data
+        ss = '\n*** Position info ***\n\n'
+        ss += '{:<20s} : {:.3f} deg\n'.format('RA (J2000)', d['RAJ2000'])
+        ss += '{:<20s} : {:.3f} deg\n'.format('Dec (J2000)', d['DEJ2000'])
+        ss += '{:<20s} : {:.3f} deg\n'.format('GLON', d['GLON'])
+        ss += '{:<20s} : {:.3f} deg\n'.format('GLAT', d['GLAT'])
 
-        ss = 'Source: {}\n'.format(d['Source_Name'])
         ss += '\n'
+        ss += '{:<20s} : {:.0f}\n'.format('ROI number', d['ROI_num'])
+        ss += '{:<20s} : {:.4f} deg\n'.format('Semimajor (68%)',
+                                              d['Conf_68_SemiMajor']
+                                              )
+        ss += '{:<20s} : {:.4f} deg\n'.format('Semiminor (68%)', d['Conf_68_SemiMinor'])
+        ss += '{:<20s} : {:.2f} deg\n'.format('Position angle (68%)', d['Conf_68_PosAng'])
 
-        ss += 'RA (J2000)  : {}\n'.format(d['RAJ2000'])
-        ss += 'Dec (J2000) : {}\n'.format(d['DEJ2000'])
-        ss += 'GLON        : {}\n'.format(d['GLON'])
-        ss += 'GLAT        : {}\n'.format(d['GLAT'])
+        ss += '{:<20s} : {:.4f} deg\n'.format('Semimajor (95%)', d['Conf_95_SemiMajor'])
+        ss += '{:<20s} : {:.4f} deg\n'.format('Semiminor (95%)', d['Conf_95_SemiMinor'])
+        ss += '{:<20s} : {:.2f} deg\n'.format('Position angle (95%)', d['Conf_95_PosAng'])
+
+        return ss
+
+    def _info_spectral(self):
+        """Print spectral info."""
+        d = self.data
+        ss = '\n*** Spectral info ***\n\n'
+        ss += '{:<45s} : {:.3} +- {:.3} erg cm^-2 s^-1\n'.format('Energy flux (100 MeV - 100 GeV)', d['Energy_Flux100'],
```

Comment:
What I usually do when the lines get too long like here is either this:
```
fmt = 'long format string'
args = ('many' ,'arguments')
ss += fmt.format(*args)
```
or this
```
ss += 'long format string'.format(
    'many', 'arguments',
    'maybe', 'many', 'more',
)
```